<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reconstructmeqt</class>
 <widget class="QMainWindow" name="reconstructmeqt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReconstructMeQt</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="scan_page">
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </widget>
      <widget class="QWidget" name="calibration_page"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionProjectHome"/>
    <addaction name="actionForum"/>
    <addaction name="actionDevice"/>
    <addaction name="separator"/>
    <addaction name="actionLog"/>
    <addaction name="actionGenerate_hardware_key"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionCalibration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="reconstruct_satus_bar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionInstallation">
   <property name="text">
    <string>Installation and Hardware</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="text">
    <string>Device Compatibility Matrix</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum</string>
   </property>
  </action>
  <action name="actionLoad_config">
   <property name="text">
    <string>load config</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save 3D model</string>
   </property>
  </action>
  <action name="actionGenerate_hardware_key">
   <property name="text">
    <string>Generate hardware hashes</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionSDKDocumentation">
   <property name="text">
    <string>SDK Documentation</string>
   </property>
  </action>
  <action name="actionProjectHome">
   <property name="text">
    <string>Project home</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../rc/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>reconstructmeqt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
